pkg/errors/error.go

```diff
@@ -38,5 +38,7 @@ var (
 	ErrUnknownValue            = errors.New("unknown value specified")
 	ErrGetFailed               = errors.New("failed to get value from the store")
 	ErrUnmarshallFailed        = errors.New("failed to unmarshall from json")
+	ErrGfproxydAlreadyEnabled  = errors.New("Gfproxyd is already enabled")
```

Comment:
This package is something I'd like to get rid of. Please keep the errors related to a package, within the package. This should be in the `gfproxyd` plugin package, in a seperate `errors.go` preferably.

Also, (again) plugins shouldn't be modifying core packages.

Reply:
Same here